### ISSUE TITLE
`Automation/2023-11-01` lacks `Delete` operation for `hybridrunbookworkergroup`

### ISSUE THREAD

**Issue by aristosvo** [hashicorp/pandora#3384]
While trying to enable [PowerShell 7.2](https://github.com/hashicorp/terraform-provider-azurerm/issues/14089) for the `automation_runbook` and `automation_module` by moving from `Automation/2022-08-08` to `Automation/2023-11-01`, I found out that the `Delete` operation for the `hybridrunbookworkergroup` client was not generated:

```
internal/services/automation/automation_hybrid_runbook_worker_group.go:184:23: client.Delete undefined (type *hybridrunbookworkergroup.HybridRunbookWorkerGroupClient has no field or method Delete)
```

It is not defined in the Data API generated by Pandora, although it is defined in the REST API specs (https://github.com/sushil490023/azure-rest-api-specs/blob/0dd49a444195fef7f3555cad038cb7665cbd928c/specification/automation/resource-manager/Microsoft.Automation/stable/2023-11-01/hybridRunbookWorkerGroup.json#L40)

**Edit1**:

The issue is probably the tag in de `delete` operation, which contains `ListAllHybridRunbookWorkerGroupInAutomationAccount` instead of `HybridRunbookWorkerGroup`.

Don't know how to fix that properly though, but at least the issue is in the schema 😟

**Edit2**:
It is a recurring issue:

- https://github.com/Azure/azure-rest-api-specs/pull/25108

At the same time another issue is still in place:

- https://github.com/Azure/azure-rest-api-specs/pull/25435

**Comment by aristosvo** [comment]
The issue is probably the tag in de `delete` operation, which contains `ListAllHybridRunbookWorkerGroupInAutomationAccount` instead of `HybridRunbookWorkerGroup`.

Don't know how to fix that properly though, but at least the issue is in the schema 😟

**Comment by aristosvo** [comment]
It is a recurring issue: 
- https://github.com/Azure/azure-rest-api-specs/pull/25108

At  the same time another issue is still in place
- https://github.com/Azure/azure-rest-api-specs/pull/25435

**Comment by aristosvo** [comment]
Fixed by #3385
